ui/admin/tests/acceptance/targets/host-sources-test.js

```diff
@@ -75,6 +87,24 @@ module('Acceptance | targets | host-sources', function (hooks) {
     assert.strictEqual(findAll('tbody tr').length, targetHostSetCount);
   });
 
+  test('can navigate to a known host set type', async function (assert) {
+    assert.expect(1);
+    await visit(urls.targetHostSources);
+    await click('main tbody .rose-table-header-cell:nth-child(1) a');
+    await a11yAudit();
+    assert.strictEqual(currentURL(), urls.hostSet);
+  });
+
+  test('cannot navigate to an unknown host set type', async function (assert) {
+    assert.expect(1);
+    instances.target.update({
+      hostSets: instances.hostCatalogPlugin.hostSets,
+    });
+
+    await visit(urls.targetHostSources);
+    assert.dom(`[href="${urls.unknownHostSet}"]`).doesNotExist();
```

Comment:
Is it possible to have the check be similar as the known host set type by looking for the actual `a` link (and it doesn't exist)?

I find it hard to verify the accuracy of tests like these because it could either be the link legitimately doesn't exist or we made a typo/got the link wrong for the `href` and we're serendipitously thinking the test is passing correctly (looks right in this case to be clear 😆).

Any thoughts on this?

Reply:
Pushed a better assert. Updated the click for the known test as well for more clarity on where we expect the user to be clicking.